packages/matter.js/src/session/SecureSession.ts

```diff
@@ -107,8 +90,10 @@ export class SecureSession<T> implements Session<T> {
             attestationKey,
             closeCallback,
             subscriptionChangedCallback,
-            idleRetransmissionTimeoutMs,
-            activeRetransmissionTimeoutMs,
+            idleIntervalMs,
```

Comment:
Maybe we could clump the intervals into a single object since they travel together.  Keys and/or other authenticated peer information might be candidates too

Reply:
Very good idea ... I adjusted it ...